apprise_api/api/templates/config.html

```diff
@@ -36,7 +36,7 @@ <h5>{% trans "Getting Started" %}</h5>
           In the future you can return to this configuration screen at any time by placing the following into your
           browser:
           {% endblocktrans %}
-          <code>{{request.scheme}}://{{request.META.HTTP_HOST}}{{BASE_URL}}/config/{{key}}</code>
+          <code>{{request.scheme}}://{{request.META.HTTP_HOST}}{{BASE_URL}}/cfg/{{key}}</code>
```

Comment:
Of your entire merge request; this is the only bug i see...  I'm not quite sure about all of the other suggested changes you made.  I've put comment on some of them, but they could almost be applied to most if not all other entries.